app/src/main/java/com/jerboa/db/AppDBMigrations.kt

```diff
@@ -321,4 +346,7 @@ val MIGRATIONS_LIST = arrayOf(
     MIGRATION_21_22,
     MIGRATION_22_21,
     MIGRATION_22_23,
+    MIGRATION_23_22,
+    MIGRATION_23_24,
+    MIGRATION_24_23,
```

Comment:
I'm testing these now, but I've never done these down migrations in room. Aren't they going to undo the previous migration, or is that all taken care of by the database version?

Reply:
Edit: seems to be taken care of by DB version.

Reply:
They wont cause a downgrade, the version on instance is what causes which version we will be using, This list is just the path the migrator can use to change between versions. If there is no path then room db throws that error on startup that says no migration found. The order of this list doesn't even matter either. Just the version numbers. I just added them because it was trivial and you never know if we have a use case for them.